tucson-teststand/calibration_lab/setup_dev_tunablelaser.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Source this file when starting the container to set it up
+echo "Running setup for environment for ts_TunableLaser CSC"
+source /home/saluser/.setup_salobj.sh
+
+setup ts_config_mtcalsys -t current
+setup ts_idl -t current
+setup ts_sal -t current
+setup ts_salobj -t current
+setup ts_tcpip -t current
+
+echo "Setting up sal, salobj and scriptqueue"
+
+echo "Running custom setups of repos"
+
+setup -j ts_config_mtcalsys -r ~/develop/ts_config_mtcalsys
+
+echo 'Installing dependencies and declaring OSPL_URI'
+pip install -e develop/ts_TunableLaser/[dev]
+export OSPL_URI=$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-shmem.xml').as_uri())")
+
+echo "# Launching ts_TunableLaser CSC"
+python /home/saluser/develop/ts_TunableLaser/bin/run_tunable_laser.py 
+# /bin/bash --rcfile /home/saluser/.bashrc
```

Comment:
Has this line been commented out on purpose? Maybe remove it if yes to avoid confusion?

Reply:
That would be a good question for @patrickingraham.

Reply:
It's there for debugging purposes as it allows me (or whomever) to get inside the container and launch it manually. I think you'll see it in a bunch of those setup_dev_CSCNAME files.
I'd rather not remove it.... and we hope to start deploying these via kubernetes in the next release (c28)

Reply:
No problem at all to leave them. I was merely curious.